modules/gitops-release-notes-1-14-0.adoc

```diff
@@ -0,0 +1,106 @@
++
+.Example 
+[source,YAML]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: argocd-sample # <1>
+  namespace: default # <2>
+spec:
+  extraConfig:
+    resource.includeEventLabelKeys: team,env* # <3>
+----
+<1> Specifies the name of the Argo CD instance.
+<2> Specifies the namespace where the Argo CD is installed.
+<3> Specifies the label keys that match and are added onto Application events.
++
+For more information, refer to link:https://argo-cd.readthedocs.io/en/latest/operator-manual/declarative-setup/#labels-on-application-events[Labels on Application Events].
+
+* With this update, users can develop applications by verifying commit signatures for `ApplicationSet` CRD created using Git generators. Before this update, users could develop applications without the necessity of verifying commit signatures. link:https://issues.redhat.com/browse/GITOPS-3049[GITOPS-3049]
++
+[NOTE]
+====
+The signature verification does not function with the templated project field in `ApplicationSet` CRD Git Generator.
+====
+
+[id="fixed-issues-1-14-0_{context}"]
+== Fixed issues
+
+* Before this update, users could not execute a rollback for multi-source applications in the Argo CD Web UI or CLI. This update fixes the issues, and users can now carry out rollbacks for multi-source applications using the Argo CD Web UI or CLI. link:https://issues.redhat.com/browse/GITOPS-3996[GITOPS-3996]
+
+* Before this update, the `ApplicationSet` Controller role did not include the `list` and `watch` permissions for the `AppProject` resource, which resulted in errors during the application generation process. This update fixes the issue by granting necessary permissions to the `ApplicationSet` Controller role. link:https://issues.redhat.com/browse/GITOPS-5401[GITOPS-5401]
+
+* Before this update, Argo CD was unable to synchronize Persistent Volume resources on {OCP} v4.16 due to the introduction of the new `.status.lastPhaseTransitionTime` field in Kubernetes v1.29. This update fixes the issue by modifying the Kubernetes static scheme in Argo CD. link:https://issues.redhat.com/browse/GITOPS-5167[GITOPS-5167]
+
+* Before this update, the {gitops-title} Operator included a race condition where the OpenShift Service CA created the TLS secret before establishing the default TLS termination policy. This resulted in `Passthrough` being set as the default policy rather than `Reencrypt`. This update fixes this race condition by verifying whether the secret was created by the OpenShift Service CA, ensuring that `Reencrypt` is used as the correct default policy. link:https://issues.redhat.com/browse/GITOPS-4947[GITOPS-4947]
```

Comment:
```suggestion
* Before this update, the {gitops-title} Operator included a race condition where the OpenShift Service CA created the TLS secret before the default TLS termination policy was set. As a result, the default policy was set as `Passthrough` instead of `Reencrypt`. This update fixes this race condition by verifying whether the secret was created by the OpenShift Service CA, ensuring that `Reencrypt` is used as the correct default policy. link:https://issues.redhat.com/browse/GITOPS-4947[GITOPS-4947]
```

Reply:
Fixed.